containers/ecr-viewer/src/app/api/save-fhir-data/route.ts

```diff
@@ -43,13 +43,21 @@ export async function POST(request: NextRequest) {
   }
 
   if (requestBody.metadata) {
-    return saveWithMetadata(
+    const saveResp = await saveWithMetadata(
       fhirBundle,
       ecrId,
       saveSource,
       requestBody.metadata,
     );
+    return NextResponse.json(
+      { message: saveResp.message },
+      { status: saveResp.status },
+    );
```

Comment:
```suggestion
    const {message, status} = await saveWithMetadata(
      fhirBundle,
      ecrId,
      saveSource,
      requestBody.metadata,
    );
    return NextResponse.json(
      { message },
      { status },
    );
```

minor style suggestion (feel free to take or leave) - can be applpied below as well